OfficeApplication/OfficeApplication/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+
+namespace OfficeApplication
+{
+    class Program
+    {
+        public  static void DepartmentWiseDisplay(string department)
+        {
+            var Context = new EmployeeEntities();
+            Console.WriteLine("Employee Id" + " | " + "Employee Name" + " | " + "Employee Address" + " | " + "Department Name" + " | " + "Salary");
+            foreach (var Employee in Context.TblEmployees)
+            {
+                if (Employee.DepartmentName == department)
+                Console.WriteLine(Employee.EmployeeId + " | " + Employee.EmployeeName + " | " + Employee.EmployeeAddress + " | " + Employee.DepartmentName + " | " + Employee.Salary);
+            }
+        }
+        public static void DisplayTotalEmployees()
+        {
+            var Context = new EmployeeEntities();
+            Console.WriteLine("Employee Id" + " | " + "Employee Name" + " | " + "Employee Address" + " | " + "Department Name" + " | " + "Salary");
+            foreach (var Employee in Context.TblEmployees)
+            Console.WriteLine(Employee.EmployeeId + " | " + Employee.EmployeeName + " | " + Employee.EmployeeAddress + " | " + Employee.DepartmentName + " | " + Employee.Salary);
+            
+        }
+        public static int GetEmployeeId()
+        {
+            Console.Write("Enter Employee Id : ");
+            int getId = Convert.ToInt32(Console.ReadLine());
```

Comment:
`  int getId = Convert.ToInt32(Console.ReadLine());
            return getId;`
Can we write like return Convert.ToInt32(Console.ReadLine());